doc/man7/EVP_ASYM_CIPHER-RSA.pod

```diff
@@ -79,7 +79,7 @@ See B<RSA_PKCS1_WITH_TLS_PADDING> on the page L<EVP_PKEY_CTX_set_rsa_padding(3)>
 
 See L<provider-asym_cipher(7)/Asymmetric Cipher Parameters> for more information.
 
-=item "pkcs15-padding-disabled" (B<OSSL_ASYM_CIPHER_PARAM_PKCS15_PADDING_DISABLED>) <integer>
+=item "rsa-pkcs15-pad-disabled" (B<OSSL_ASYM_CIPHER_PARAM_FIPS_RSA_PKCS15_PAD_DISABLED>) <integer>
```

Comment:
Can we change this name in a minor release?  I.e since OSSL_ASYM_CIPHER_PARAM_PKCS15_PADDING_DISABLED already exists would this be considered an ABI breakage?

Reply:
@paulidale  did you backport this particular name to other branches for testing purposes?

Reply:
I'm pretty sure I didn't.  I don't see the define in the older branches.

Reply:
I couldnt see it either, and this is a new item added just to master, so it should be ok to change.